### ISSUE TITLE
Live server is not possible without body or head tag

### ISSUE THREAD

**Issue by Chynweeme** [ritwickdey/vscode-live-server#2419]
HI! I tried running live server on my vscode today and I keep getting the prompt "live server is not possible without body or head tag. I ran the same live server yesterday and it worked fine. What can be done about it please?

## I'm submitting a...

```html
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
[ ] Other: <!-- Please describe: -->
```

## Environment

```html
Browser:

- [ ] Chrome (desktop) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] IE version XX
- [ ] Edge version XX

For Tooling issues:

- Live Server: <!-- Check which version is installed --> 
- Platform:  <!-- Mac, Linux, Windows -->
- Visual Studio Code: <!-- `code --version` -->
```

**Comment by DerickReyes** [comment]
bro
I have the same problem, when the document is saved, this error appears and the pages are not refreshed. Tell me, were you able to solve the problem?

And if you could please help me, I need to be able to use that extension as before.

The Live Server extension has lousy support.

**Comment by ritwickdey** [comment]
Please create a file and paste this HTML.
```html
<!DOCTYPE html>
<html lang="en">
  <head></head>
  <body>
    Hello World
  </body>
</html>
```

Now try to open the file using Live Server.  It should not give that warning message.

**Comment by ritwickdey** [comment]
No, you have to add `Body` (don't translate.) tag.

Example:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/18099464/166940842-da2cf91f-11c9-438d-ab3a-e55690298c01.png">

**Comment by DerickReyes** [comment]
> No, debe agregar `Body`(no traducir) la etiqueta.
> 
> Ejemplo: <img alt="imagen" width="412" src="https://user-images.githubusercontent.com/18099464/166940842-da2cf91f-11c9-438d-ab3a-e55690298c01.png">

Yes
I use that and nothing, I think the error is in the configuration of the Live Server extension in the area of: settings.json.

You can contact me by call or give me a number to call and explain better. I speak the Latin Spanish language.

**Comment by DerickReyes** [comment]
> @DerickReyes
> 
> ¿Puedes probar este (sin traducir)?
> 
> ```
> <html lang="en">
>   <head></head>
>   <body>
>     Hello World
>   </body>
> </html>
> ```
> 
> Y abra esto usando Live Server. y comparta la captura de pantalla del navegador y la ventana de Vscode (completa).
It's just that I don't see the option to send images in this chat or I don't know how to send it.

Another thing. Live Server, Sometimes it works for me, but after a while it stops working, and always with that routine, when it works for a while and then it stops working.

**Comment by ritwickdey** [comment]
I mean just a normal screenshot. Please search on google how you can take a screenshot of a full window.

If you're using Mac, then it's `cmd+shift+3`.

then you can directly drag the screenshot into this comment.

> box.

**Comment by DerickReyes** [comment]
> Me refiero a una captura de pantalla normal. Busque en Google cómo puede tomar una captura de pantalla de una ventana completa.
> 
> Si está usando Mac, entonces es `cmd+shift+3`.
> 
> luego puede arrastrar directamente la captura de pantalla a este cuadro de comentarios.

ok

**Comment by DerickReyes** [comment]
> Me refiero a una captura de pantalla normal. Por favor busque en google. Si está usando Mac, entonces es `cmd+shift+3`.
> 
> Y puede arrastrar directamente la captura de pantalla a este cuadro de comentarios.

![e1](https://user-images.githubusercontent.com/104990540/166946029-506537df-b787-45b4-8a5c-9801356b4cc5.PNG)


Still it keeps giving me the error.

**Comment by DerickReyes** [comment]
![e2](https://user-images.githubusercontent.com/104990540/166946842-96ea8bd1-225a-4ed2-964c-70bd24b2884a.PNG)

I think my error is in the settings: (settings.json).
LiveServer

**Comment by DerickReyes** [comment]
> Weird. Let me know how I can reach you (Gmail id)?

derickjhoelv@gmail.com

**Comment by ritwickdey** [comment]
[INFO/Workaround]

After debugging with @DerickReyes, I found if you're not using the English language then, specify the language code into the HTML.

@DerickReyes was using Spanish. After setting `<html lang="es">` this, the problem solved. 

I think it's an encoding bug. I've to check.

**Comment by DerickReyes** [comment]
> [INFORMACIÓN/Solución alternativa]
> 
> Después de depurar con@DerickReyes, encontré que si no está usando el idioma inglés, especifique el código de idioma en el HTML.
> 
> @DerickReyesestaba usando el español. Después de configurar `<html lang="es">`esto, el problema se resolvió.
> 
> Creo que es un error de codificación. tengo que comprobar.

The error continues even if it is in "ES".

**Comment by Chynweeme** [comment]
> Please create a file and paste this HTML.
> 
> ```
> <!DOCTYPE html>
> <html lang="en">
>   <head></head>
>   <body>
>     Hello World
>   </body>
> </html>
> ```
> 
> Now try to open the file using Live Server. It should not give that warning message.



> 

I've been able to resolve it.  I discovered that the file I was trying to run on the line server want saved on my VS code.. this was surprising because I opened the same file on line server the previous day.. anyways I saved it again and was able to open it on live server finally. 
Thank you @ritwickdey and others for your help. It's well appreciated..

**Comment by DerickReyes** [comment]
> > Please create a file and paste this HTML.
> > ```
> > <!DOCTYPE html>
> > <html lang="en">
> >   <head></head>
> >   <body>
> >     Hello World
> >   </body>
> > </html>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Now try to open the file using Live Server. It should not give that warning message.
> 
> > 
> 
> I've been able to resolve it. I discovered that the file I was trying to run on the line server want saved on my VS code.. this was surprising because I opened the same file on line server the previous day.. anyways I saved it again and was able to open it on live server finally. Thank you @ritwickdey and others for your help. It's well appreciated..

help

**Comment by DerickReyes** [comment]
Hello. I need help to solve that above problem. I have already done all I could and my reach. I even remove the extension and reinstall it and the error is still there, and I changed the operating system on my computer and the error is still there. 

The error started to appear after I modified the configuration of the .jso document. I modified it to say that the document will save automatically and sometimes it works and sometimes it doesn't, where it doesn't work is where I get the error. The fellow @ritwickdey. He was able to see the situation I'm going through with the Live Server Extension..... I hope someone can help me and the error is solved. With this im out.

**Comment by DerickReyes** [comment]
Hello. I need help to solve that above problem. I have already done all I could and my reach. I even remove the extension and reinstall it and the error is still there, and I changed the operating system on my computer and the error is still there. 

The error started to appear after I modified the configuration of the .jso document. I modified it to say that the document will save automatically and sometimes it works and sometimes it doesn't, where it doesn't work is where I get the error. The fellow @ritwickdey. He was able to see the situation I'm going through with the Live Server Extension..... I hope someone can help me and the error is solved. With this im out.
